H2MLauncher.UI/MainWindow.xaml

```diff
@@ -46,6 +46,7 @@
             </Style.Triggers>
         </Style>
         <local:HostNameColorConverter x:Key="hostNameColorConverter" />
+        <local:PingColorConverter x:Key="PingColorConverter"/>
```

Comment:
We named the other converter `hostNameColorConverter` I think we can stick to that same style, so do `pingColorConverter`

Reply:
![image](https://github.com/user-attachments/assets/0b90a2e7-c09c-454e-ac8a-43cfd17c54b0)
I did it uppercase because of suggestion

Reply:
Ah, ok. Then you might as well update the other one.